docs/source/refs/reference_architecture/index.rst

```diff
@@ -0,0 +1,361 @@
+Reference Architecture
+====================================
+
+This document presents an overview of the end-to-end robot learning process with
+Isaac Lab and Isaac Sim. This is demonstrated using a reference architecture that highlights
+the major building blocks for training and deployment workflows. It provides a comprehensive,
+user-friendly guide on the entire process of developing applications from training to deploying
+the trained model in the real world, including links to demos, working examples, and documentation.
+
+Who is this document for?
+---------------------------------
+
+This document is designed to assist robotics developers and researchers working with NVIDIA Isaac Lab
+in the robot learning field, including those at research labs, Original Equipment Manufacturers (OEM),
+Solutions Providers, Solutions Integrators (SI),  and independent software vendors (ISV). It offers
+guidance on utilizing Isaac Lab’s robot training framework and workflows as a foundational starting
+point for environment configuration, task design, and policy training and testing.
+
+
+
+.. image:: ../../_static/reference-architecture/isaac-lab-ra-light.svg
+    :class: only-light
+    :align: center
+    :alt: Isaac Lab Reference Architecture
+
+.. image:: ../../_static/reference-architecture/isaac-lab-ra-dark.svg
+    :class: only-dark
+    :align: center
+    :alt: Isaac Lab Reference Architecture
+
+
+|
+
+The reference architecture for Isaac Lab comprises the following components:
+
+1. **Asset Input**
+2. **Configuration - Assets & Scene**
+3. **Robot Learning Task Design**
+4. **Register with Gymnasium (Optional)**
+5. **Environment Wrapping**
+6. **Run Training**
+
+  * Single GPU Training
+  * Multi-GPU and Multi-Node Training
+  * Cloud-based Training
+
+7. **Run Testing**
+
+
+**Components**
```

Comment:
Are all the bolded titles necessary? would be nice to keep the styling consistent.

Reply:
Updated